Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM node:16.11
+
+WORKDIR /usr/src/app
+
+COPY / ./
```

Comment:
is there any improvement you could have done here to improve build time when rebuilding this image?

Reply:
To improve the build time, we can only copy the package.json or package-lock.json files like following line
`COPY package.json package-lock.json ./`